pavelib/assets.py

```diff
@@ -20,20 +20,34 @@
 
 ALL_SYSTEMS = ['lms', 'studio']
 COFFEE_DIRS = ['lms', 'cms', 'common']
+
 # A list of directories.  Each will be paired with a sibling /css directory.
 COMMON_SASS_DIRECTORIES = [
-    path("common/static/sass"),
+    (path("common/static/sass"), path("common/static/css"))
+]
+COMMON_SASS_LOOKUP_DIRECTORIES = [
+    'common/static',
+    'common/static/sass'
 ]
+
 LMS_SASS_DIRECTORIES = [
-    path("lms/static/sass"),
-    path("lms/static/themed_sass"),
-    path("lms/static/certificates/sass"),
+    (path("lms/static/sass"), path("lms/static/css")),
+    (path("lms/static/themed_sass"), ""),
+    (path("lms/static/certificates/sass"), path("lms/static/certificates/css")),
+]
+LMS_SASS_LOOKUP_DIRECTORIES = [
+    path("lms/static/sass/partials"),
 ]
+
 CMS_SASS_DIRECTORIES = [
-    path("cms/static/sass"),
+    (path("cms/static/sass"), path("cms/static/css")),
+]
+CMS_SASS_LOOKUP_DIRECTORIES = [
+    path("cms/static/sass/partials"),
 ]
+
 THEME_SASS_DIRECTORIES = []
-SASS_LOAD_PATHS = ['common/static', 'common/static/sass']
+THEME_SASS_LOOKUP_DIRECTORIES = []
```

Comment:
I think we'll have to split `THEME_SASS_DIRECTORIES` into `THEME_LMS_SASS_DIRECTORIES` and `THEME_CMS_SASS_DIRECTORIES`; and `THEME_SASS_LOOKUP_DIRECTORIES` into `THEME_SASS_LMS_LOOKUP_DIRECTORIES` and `THEME_SASS_CMS_LOOKUP_DIRECTORIES`.

Currently it doesn't matter much because most of the time we compile all sass files in a single command (`systems` equals `ALL_SYSTEMS`), but if we decide to move more sass files to the `partials` directory, we will quickly start to get name clashes between partials in the lms partials folder and partials in the cms partials folder, so it will be important to compile lms and cms sass in two separate commands.